lib/views/pages/professional/professional_page.dart

```diff
@@ -1,23 +1,36 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:withtone/providers/user/login_user.dart';
 import 'package:withtone/views/components/admob/ad_banner.dart';
-import 'package:withtone/views/components/circle_icon_button.dart';
-import 'package:withtone/views/components/circle_profile_icon.dart';
+import 'package:withtone/views/components/icon/circle_icon_button.dart';
+import 'package:withtone/views/components/icon/circle_profile_icon.dart';
 import 'package:withtone/views/pages/professional/background_color.dart';
 import 'package:withtone/views/pages/professional/professional_tile.dart';
 
 ///プロフェッショナルページ
-class ProfessionalPage extends StatefulWidget {
+class ProfessionalPage extends ConsumerStatefulWidget {
   const ProfessionalPage({super.key});
 
   static const String path = '/professional';
 
   @override
-  State<ProfessionalPage> createState() => _ProfessionalPageState();
+  ConsumerState<ProfessionalPage> createState() => _ProfessionalPageState();
 }
 
-class _ProfessionalPageState extends State<ProfessionalPage> {
+class _ProfessionalPageState extends ConsumerState<ProfessionalPage> {
   @override
   Widget build(BuildContext context) {
+    // loginUser (AsyncValue<WithToneUser>) を取得
+    final loginUser = ref.watch(loginUserProvider);
+
+    // loginUser.userImage から Widget を生成
+    final userIcon = switch (loginUser) {
+      AsyncError() => const Icon(Icons.person),
+      AsyncData(:final value) => CircleProfileIcon(
+          size: 44, isIcon: false, imageUrl: value.userImage, onPressed: () {}),
```

Comment:
```suggestion
          size: 44, isIcon: false, imageUrl: value.userImage, onPressed: () => Navigator.pushNamed(context, ProfilePage.path),),
```
こちらもプロフィールに飛ばしていいのかなと思いましたが、どうでしょうか！

Reply:
対応しました！